api/renter.go

```diff
@@ -29,7 +31,6 @@ type FileInfo struct {
 	UploadProgress float32
```

Comment:
as opposed to...? Precision isn't crucial here, but making it a float64 wouldn't hurt.